packages/javascript/js-client-library/src/responses.ts

```diff
@@ -100,6 +100,25 @@ type PostureStat = TimestampFields & {
 
 export type PostureResponse = PaginatedResponse<PostureStat[]>;
 
+type PostureFindingTrend = {
+    finding: string;
+    start_count: number;
+    end_count: number;
+    severity: string;
```

Comment:
I think this would likely need to be a number - or we would want two properties `severity` and `severity_label` perhaps